### DISCUSSION TITLE
Howto use GridEntityCamera/PixelSmoothCamera to point fix to an entity

### DISCUSSION THREAD

**Discussion by jobe-m** [littlektframework/littlekt#217] [Q&A] [answered]
Hi lehaine,

you created a very nice framework here. I am looking at it for couple of weeks already and I really like it a lot.

Now I would like to use a camera to follow an entity on the screen. That camera should always place the entity in the middle of the screen. Thus I don't want the camera to have any kind of friction, but instantly follow the entity.

I set `friction` of `GridEntityCamera` to `0`, but that did not do the trick. The entity still can be moved "out" of the central position on the screen. Or do I need to use `PixelSmoothCamera` and set there the `offset` or do I need to directly set the `position` with `translate()` in the `Camera` base class?

**Comment by LeHaine** [comment, marked as answer]
Hey!

The `GridEntityCamera` uses a "deadzone" that prevents the camera from moving if the character moves slightly. We can adjust this to always be centered by setting `deadZonePctX` and `deadZonePctY` to `0`.

**Reply by jobe-m** [reply]
Thanks, that works :)
